SDK/MobileCenter/Microsoft.Azure.Mobile.UWP/MobileCenterPart.cs

```diff
@@ -1,7 +1,18 @@
-﻿namespace Microsoft.Azure.Mobile
+﻿using Microsoft.Azure.Mobile.Utils;
+
+namespace Microsoft.Azure.Mobile
 {
     public partial class MobileCenter
     {
         private const string PlatformIdentifier = "uwp";
+
+        /// <summary>
+        /// Sets the device country to display in the portal
```

Comment:
Well since the UWP country is not tied to the carrier, it didn't make sense to refer to it as "carrier country"